pom.xml

```diff
@@ -502,6 +502,8 @@
           <format>OCI</format>
           <creationTime>USE_CURRENT_TIMESTAMP</creationTime>
           <entrypoint>${cryostat.entrypoint}</entrypoint>
+          <mainClass>io.cryostat.Cryostat</mainClass>
```

Comment:
There's also already a `mainClass` reference in the `exec-maven-plugin` configuration. Could this value be extracted to a property and referenced in both of these instances?